addon/i3dio/ui/collision_data.py

```diff
@@ -223,6 +222,8 @@ def convert_old_collision_masks(dummy) -> None:
             obj.i3d_attributes.collision_filter_group = result['group_hex']
             obj.i3d_attributes.collision_filter_mask = result['mask_hex']
 
+            del obj.i3d_attributes['collision_mask']
```

Comment:
Is it necessary to explicitely delete it? I think Blender will delete it automatically on next save as the property isn't registered anymore and thus wont be able to be written into the savefile.

Reply:
Did some more testing now, it seems to stay there even after you save and reopen the file